drake/Makefile

```diff
@@ -33,13 +33,9 @@ endif
 
 .PHONY: clean
 clean:
-	cmake --build pod-build --target clean
-# for windows cmd shell
-	@-rd /s pod-build        
-	@-rd /s build
-# for everyone else
-	@-rm -rf pod-build
-	@-rm -rf build
+	cmake --build pod-build --target clean-all
```

Comment:
Maybe I don't understand, but I thought the target should be clean, not clean-all. I don't think the CMakeLists.txt in drake-distro/drake defines a clean-all target. Same for all of the externals.